mantidproject/mantid
# Indirect - Plot Spectrum and Tiled Plot options

**Description of work.**
This PR introduces options to choose which result spectrum to plot in the `I(Q,t)` Tab of Indirect Data Analysis. It also allows you to choose a range of spectra which you want to plot using Tiled Plot, up to a maximum of 18 spectra.

**To test:**
1. `Interfaces`->`Indirect Data Analysis`->`I(Q,t)` Tab
2. Load in the data files below
3. Click `Run` and wait
4. Choose which spectrum you would like to plot and click `Plot Spectrum`. Each spectrum is likely to be slightly different (this is more noticeable to the right of the plot).
5. Select a range of spectra for the tiled plot. Click `Tiled Plot`, and the correct spectra should be plotted.

- Plot Spectrum Spin box max and min for this data set is 50 and 0
- The range of spectra for tiled plot should only allow you to cover a maximum range of 18 (i.e. 0 to 17, 1 to 18, 5 to 22, etc.)

**Data**
[irs26173_graphite002_res.zip](https://github.com/mantidproject/mantid/files/2456102/irs26173_graphite002_res.zip)
[irs26176_graphite002_red.zip](https://github.com/mantidproject/mantid/files/2456103/irs26176_graphite002_red.zip)

Fixes #23543 and #23617 

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
